react/Logo.tsx

```diff
@@ -2,30 +2,33 @@ import React from 'react'
 import { Link, useRuntime } from 'vtex.render-runtime'
 import type { RenderContext } from 'vtex.render-runtime'
 import { useCssHandles } from 'vtex.css-handles'
+import type { CssHandlesTypes } from 'vtex.css-handles'
 import * as Amp from 'react-amphtml'
 
 import Placeholder from './components/Logo/LogoPlaceholder'
 import styles from './components/Logo/styles.css'
 
+const CSS_HANDLES = ['logoLink', 'logoImage', 'logoContainer'] as const
+
 interface Props {
   /** Logo's URL */
-  url: string
+  url?: string
```

Comment:
Why should this be optional? I know we do handle that case, so the component doesn't crash. But I think that's more of a protection. I think it makes sense for the `url` to be required